game/scripts/vscripts/Sections/Chapter5/SectionRoshan.ts

```diff
@@ -0,0 +1,270 @@
+import * as tut from "../../Tutorial/Core";
+import * as tg from "../../TutorialGraph/index";
+import { RequiredState } from "../../Tutorial/RequiredState";
+import { GoalTracker } from "../../Goals";
+import { chapter5Blockades, runeSpawnsLocations } from "./Shared";
+import { findRealPlayerID, getOrError, getPlayerHero, setUnitPacifist } from "../../util";
+import { modifier_custom_roshan_attack_speed } from "../../modifiers/modifier_custom_roshan_attack_speed";
+
+const sectionName: SectionName = SectionName.Chapter5_Roshan;
+
+let graph: tg.TutorialStep | undefined = undefined
+let canPlayerIssueOrders = false;
+
+const requiredState: RequiredState = {
+    requireSlacksGolem: true,
```

Comment:
Should add heroLevel: 6 and ability minimum [1, 1, 1, 1] to requiredState

Reply:
Is this from one of the previous chapters? Think I have to add it to the opening section too then (for that section I can do it in later PR since I have to update it anyway)